spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:example) { get '/users/1/posts' }
+
+    it 'displays a list of posts' do
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template('index')
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      expect(response.body).to include('Here is a list of posts')
+    end
+
+    describe 'GET /show', type: :request do
+      before(:example) { get '/users/1/posts/1' }
+      it 'displays the post details for a given post' do
+        expect(response).to have_http_status(200)
```

Comment:
- [ ] Half of all your tests are failing. Kindly fix them.

  ![image](https://github.com/yashodhicy/Blog-App/assets/81468675/c1b72af2-61a2-4026-8f24-a3e1e9e9891d)

Reply:
Can you give me specific reasons why they are failing?  Because from my side those are passing. @julie-ify 
![image](https://github.com/yashodhicy/Blog-App/assets/116276432/07751148-bd62-4dda-8220-62e17cd096b6)